client/question/services/FeedbackGeneratorService.js

```diff
@@ -803,6 +803,14 @@ tie.factory('FeedbackGeneratorService', [
                   feedback.appendErrorParagraph(paragraph.content);
                 }
               });
+
+              var errorLineNumber = prereqCheckFailure.getErrorLineNumber();
```

Comment:
Interesting that this is in the forEach loop. If a piece of code exhibits multiple such errors, do we show all of them? If so, there would be different error line numbers for each. Would it make more sense to show just the first one?

Reply:
This lives inside of the hasWrongLanguage Prereq check error loop. Line errors are only captured this way for hasWrongLanguage Errors. The loop breaks after finding the first error.

Syntax errors and line numbers are currently in the spoiler link popout. It wasn't clear if they should be added as feedback bubbles at this time.

Reply:
I think what I mean is, should this also break once we've successfully set an error line number and appended a text paragraph? Looking at this code in isolation suggests that the error line number that is set corresponds to the latest one found, and there may be one or more text paragraphs containing error strings.